pom.xml

```diff
@@ -1,360 +1,278 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-	<modelVersion>4.0.0</modelVersion>
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

Comment:
could you please revert this unrelated changes?

Reply:
The IDE did this without notice, reverting...

Reply:
@bsideup  Changes done (i had to do a force-push), waiting your review, thank you.